site/components/form/MapBox.tsx

```diff
@@ -0,0 +1,23 @@
+import { CSSProperties, ReactElement } from "react";
+import Map, { ViewState } from "react-map-gl";
+
+interface MapBoxProps {
+    initialViewState: Partial<ViewState>;
+    style: CSSProperties;
+    mapStyle: string;
+}
+
+const MapBox = ({ initialViewState, style, mapStyle }: MapBoxProps): ReactElement => {
+    const mapboxAccessToken = process.env.MAP_BOX_ACCESS_TOKEN || "";
+
+    return (
```

Comment:
We probably only want to try and render the map if the mapboxAccessToken is non empty

Reply:
done